pytest_inmanta/test_parameter/path_parameter.py

```diff
@@ -25,18 +24,11 @@
 from pytest_inmanta.test_parameter.parameter import DynamicDefault, TestParameter
 
 
-@contextlib.contextmanager
-def working_dir(path: str):
+def abspath(path: str, *, source: str) -> str:
```

Comment:
I don't see particular value in having a dedicated method for this, but I'm not against. it. If we do, the `source` should have a `:param:` in the docstring, because it's not trivial.